test/suite/commands/openFromUrl.test.ts

```diff
@@ -150,8 +157,31 @@ describe("openFromUrl.ts", async () => {
       showTextDocumentStub.resolves();
 
       await openWorkspace(manifestUri.fsPath);
-      expect(executeCommandStub.called).to.be.true;
-      expect(showTextDocumentStub.called).to.be.true;
+      expect(executeCommandStub.calledOnceWith("vscode.openFolder")).to.be.true;
     });
   });
+
+  describe("getAddonByUrl", async () => {
+    it("should receive a result from downloadAndExtract and correctly call openWorkspace", async () => {
```

Comment:
Make sure new tests have a terminal period as adopted in #46.

```suggestion
    it("should receive a result from downloadAndExtract and correctly call openWorkspace.", async () => {
```